### PR TITLE
Add reference to context in storage service HTTP apps

### PR DESCRIPTION
**Pull request by alicefuzier**
For #2906

### CODE REVIEW

File: archive/archive_api/src/responses.py [hunk 1]
```diff
@@ -35,23 +33,12 @@ def __init__(self, response, *args, **kwargs):
         # The @context may already be provided if we've been through the
         # force_type method below.  We also don't add a context if we're
         # looking at the healthcheck endpoint.
-        if ("@context" in rv) or (rv == {"status": "OK"}):
-            return super().__init__(response, *args, **kwargs)
-        else:
-            rv["@context"] = self.context_url
-            json_string = json.dumps(rv)
-            return super().__init__(json_string, *args, **kwargs)
+        return super().__init__(response, *args, **kwargs)
```
Comment (alexwlchan) [inline archive/archive_api/src/responses.py]: At this point I suspect we can ditch this entire class.

File: archive/registrar_http/src/test/scala/uk/ac/wellcome/platform/archive/registrar/http/RegistrarHttpFeatureTest.scala [hunk 2]
```diff
@@ -103,6 +104,8 @@ class RegistrarHttpFeatureTest
                           "Location"),
                         createdDateString,
                         "Bag") =>
+
+                      actualContectUrl shouldBe "http://api.wellcomecollection.org/storage/v1/context.json"
```
Comment (alexwlchan) [inline archive/registrar_http/src/test/scala/uk/ac/wellcome/platform/archive/registrar/http/RegistrarHttpFeatureTest.scala]:
```suggestion
                      actualContextUrl shouldBe "http://api.wellcomecollection.org/storage/v1/context.json"
```

File: archive/registrar_http/src/test/scala/uk/ac/wellcome/platform/archive/registrar/http/RegistrarHttpFeatureTest.scala [hunk 3]
```diff
@@ -81,6 +81,7 @@ class RegistrarHttpFeatureTest
 
                   inside(displayBag) {
                     case DisplayBag(
+                      actualContectUrl,
```
Comment (alexwlchan) [inline archive/registrar_http/src/test/scala/uk/ac/wellcome/platform/archive/registrar/http/RegistrarHttpFeatureTest.scala]:
```suggestion
                      actualContextUrl,
```

File: archive/registrar_http/src/test/scala/uk/ac/wellcome/platform/archive/registrar/http/RegistrarHttpFeatureTest.scala (hunk 2 above)
Comment (alicefuzier) [reply archive/registrar_http/src/test/scala/uk/ac/wellcome/platform/archive/registrar/http/RegistrarHttpFeatureTest.scala]: oops

File: archive/archive_api/src/responses.py (hunk 1 above)
Comment (alicefuzier) [reply archive/archive_api/src/responses.py]: I think so but I couldn't figure out how to do that and I couldn't be bothered anyway cause all this code is going away entirely. We still have very out of date models in the models folder for example

**Comment by alicefuzier** [comment]
This is deployed and tested both via the archive_api route and the api gateway route and it works fine

**Pull request merged**